CHANGELOG.md

```diff
@@ -4,11 +4,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.18.0] - 2022-03-02
+
+### Added
+
+- Filter one or multiple events by type from an EventSubscription ([#461](https://github.com/paritytech/subxt/pull/461))
+- Distinct handling for N fields + 1 hasher vs N fields + N hashers ([#458](https://github.com/paritytech/subxt/pull/458))
+- Add hardcoded override to ElectionScore ([#455](https://github.com/paritytech/subxt/pull/455))
+- Expose method to fetch nonce via `Client` ([#451](https://github.com/paritytech/subxt/pull/451))
+- New Event Subscription API ([#442](https://github.com/paritytech/subxt/pull/442))
```

Comment:
This replaced the old event subscription API, so probably a Change rather than an add too :)

I'd probably bundle this together with #461, above, since they both are part of this new event subscription api :)